appveyor.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.0.{build}
+environment:
+  matrix:
+    - nodejs_version: "0.8"
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.12"
+    # fails due to iojs bug: - nodejs_version: "1.8"
+    - nodejs_version: "2.5.0"
+    - nodejs_version: "3.3.0"
+    - nodejs_version: "4"
+    - nodejs_version: "5"
+os:
+  - Visual Studio 2013
+  - Visual Studio 2015
+install:
+  - ps: Install-Product node $env:nodejs_version x64
+  # Sets Windows 7.1 SDK env vars.
+  - '"C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64'
+  # Prepend 2015 and 2013 tools. Harmless when the path for the other version doesn't exist.
+  - set "PATH=%ProgramFiles(x86)%\MSBuild\14.0\Bin;%ProgramFiles(x86)%\MSBuild\12.0\Bin;%PATH%"
+  # Upgrade npm to latest
+  - npm install --loglevel error -g npm 
+  - set "PATH=%APPDATA%\npm;%PATH%"
+  - node -v
+  - npm -v
+  - npm install -g --loglevel error node-gyp
+  # Put GTK in C:/
+  - curl -fLsS -o 'gtk.zip' 'http://ftp.gnome.org/pub/GNOME/binaries/win64/gtk+/2.22/gtk+-bundle_2.22.1-20101229_win64.zip'
+  - 7z x gtk.zip -oGTK > nul
+  - mv GTK/ C:/
+  - npm install
+build: off
```

Comment:
No, the default build command invokes msbuild directly. Getting the configuration right for that looks more verbose/complicated than it currently is.